docker-compose.yaml

```diff
@@ -13,7 +13,8 @@ services:
 
   api-gateway:
     build:
-      context: api_gateway
+      context: .
```

Comment:
why the context switching?

Reply:
Just for consistence, as the catalog is also using the root directory as its context.